menuv.lua

```diff
@@ -184,6 +184,40 @@ function MenuV:CreateMenu(title, subtitle, position, r, g, b, size, texture, dic
     return self.Menus[index] or menu
 end
 
+--- Create a menu that inherits properties from another menu
+---@param parent Menu|string Menu or UUID of menu
+---@param overrides table<string, string|number> Properties to override in menu object (ignore parent)
+function MenuV:CreateChildMenu(parent, overrides)
```

Comment:
Same goes for this one:

`CreateChildMenu` should be `CloneMenu` or `InheritMenu` because child menu's don't exist and each menu is equal to each other.